remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/ConfigCache.kt

```diff
@@ -60,6 +59,13 @@ internal class ConfigCache @VisibleForTesting constructor(
 
     fun getConfig(): Map<String, String> = configBody
 
+    suspend fun fetchAndApplyConfig() = withContext(dispatcher) {
+        fetchConfig()
+        configBody = applyConfig()
+        poller.reset() // stop current polled fetch, then restart after interval.
```

Comment:
Maybe the poller should be reset before fetching the config? Otherwise, we could hit a situation where the poller tries to fetch while a manual fetch is in progress.

Reply:
separated stop and restart